cloud/ec2.go

```diff
@@ -1059,6 +1063,18 @@ func (m *ec2Manager) OnUp(ctx context.Context, h *host.Host) error {
 	}
 	defer m.client.Close()
 
+	// Add instance stores and volumes to host document
+	devices, err := m.client.GetInstanceBlockDevices(ctx, h)
```

Comment:
Instead of doing this, can we also save the block device mappings when we get the DNS name? It looks like they are return in the same API call

Reply:
Looks like Thomas already thought of this. He saved it when GetInstanceStatus is called, so I think you can remove the API call here